lib/src/buttons/src/action_button.dart

```diff
@@ -119,12 +126,20 @@ class _ThemedActionsButtonsState extends State<ThemedActionsButtons> with Single
 
     double width = 150;
 
-    double? right = screenSize.width - offset.dx - boxSize.width;
+    double? right = screenSize.width - offset.dx - boxSize.width + actionsOffset.dx;
     double? left;
 
+    double? top = offset.dy + actionsOffset.dy;
```

Comment:
@XenonXe1705 I applied some changes here to handle bottom and top re-location using your actionOffset, your code only consider the left and right reposition, with this change that should work in basically any scenario.

Good job in the left and right condition.